### DISCUSSION TITLE
sup fam, welcome to steve discussions!

### DISCUSSION THREAD

**Discussion by jwford** [stevebot-project/steve#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BenSegal855** [comment]
Hello fellow Steveians. I’m Ben! I used Steve to learn js and ts. Since then, I have forked Steve and used him as a codebase for my universities discord bot. We’ll be moving to a custom codebase in the future but Steve literally got me a job so that’s kinda awesome. I don’t plan on stopping working on Steve though. I have a few lines of code in the current codebase like the discord status command and role ping injection detection. I plan to keep the PRs coming with more cool stuff that pops into my head. 

Don’t need to know where I’m going, just need to know where I’ve,
Ben

**Reply by colinkautz** [reply]
10/10 for that name specific sign-off

**Reply by BenSegal855** [reply]
I've had that for a while

**Comment by colinkautz** [comment]
Hey, I'm Colin. I work for a shoe company doing front-end web development. I've used the Steve repo to learn about DiscordJS and to sharpen my Typescript skills. I hope I can help with more bugs/features down the road
